src/pymediainfo/__init__.py

```diff
@@ -163,15 +163,12 @@ def __eq__(self, other: object) -> bool:
         return self.tracks == other.tracks
 
     def __init__(self, xml: str, encoding_errors: str = "strict") -> None:
-        xml_dom = ET.fromstring(xml.encode("utf-8", encoding_errors))
+        xml_dom = ET.fromstring(xml.encode("utf-8", encoding_errors))  # noqa: S314
```

Comment:
ruff recommendation is to use the `defusedxml` module instead of the stdlib because [it corrects several vulnerabilities](https://pypi.org/project/defusedxml).
But it would add a dependency to the project.